homeassistant/components/tahoma/cover.py

```diff
@@ -223,16 +224,16 @@ def stop_cover(self, **kwargs):
         ):
             self.apply_action("setPosition", "secured")
         elif self.tahoma_device.type in (
```

Comment:
If the container should hold unique items we should make it a set instead of a tuple.

Reply:
Done